src/main/java/ru/sbt/mipt/oop/main/Application.java

```diff
@@ -13,14 +14,14 @@ public Application(SmartHomeGettable smartHomeGettable) {
     }
 
     public static void main(String... args) throws IOException {
-        List<DoorsAndLightsManager> allDoorsAndLightsManager = new ArrayList<>();
-        allDoorsAndLightsManager.add(new LightEventManager());
-        allDoorsAndLightsManager.add(new DoorEventManager());
-        allDoorsAndLightsManager.add(new HallDoorEventManager());
+        List<EventHandler> allEventHandler = new ArrayList<>(Arrays.asList(new AlarmManager(),
```

Comment:
https://github.com/iagoshkov/smart-home-2020/blob/master/STUDENT_GUIDE.md#%D1%81%D0%B4%D0%B5%D0%BB%D0%B0%D0%B9%D1%82%D0%B5-%D0%BE%D1%82%D0%BF%D1%80%D0%B0%D0%B2%D0%BA%D1%83-%D0%BC%D0%B0%D0%BA%D1%81%D0%B8%D0%BC%D1%83%D0%BC-%D0%BE%D0%B4%D0%BD%D0%BE%D0%B3%D0%BE-%D1%81%D0%BC%D1%81-%D0%B8%D0%B7-%D0%B4%D0%B5%D0%BA%D0%BE%D1%80%D0%B0%D1%82%D0%BE%D1%80%D0%B0